packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/helpers/isFollowedByExceptionSpec.js

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line max-len
+import { isFollowedByException, isNotFollowedByException } from "../../../../../src/scoring/assessments/inclusiveLanguage/helpers/isFollowedByException";
+
+describe( "Test isFollowedByException and isFollowedByException", () => {
```

Comment:
I would split this up into two `describe` statements. One for tests covering the `isFollowedByException` function and one for tests covering the `isNotFollowedByException` function.

Reply:
The reasoning is that with each `describe` statement we are describing the expected behavior of a single unit in the code. In this case I would say that the units we want to describe are the individual functions.

Reply:
You are right.
The reason why I thought an exception was justified, is that combining them would lead to a large reduction in lines of code.